.editorconfig

```diff
@@ -7,4 +7,4 @@ indent_style = space
 indent_size = 2
 
 [*.erb]
-indent_size = 4
+indent_size = 2
```

Comment:
Also, if we're going to change this then just delete this because the above line says the same thing?